duckduckgo/Android
# Fix UI when a site is in temp allowed list

Task/Issue URL: https://app.asana.com/0/inbox/1125189844075764/1200517965702749/1200538500897521
Tech Design URL: 
CC: 

**Description**:
This PR changes the privacy dashboard UI when a visited site is in the temporary allowed list to correctly show that Privacy Protections are disabled.

**Steps to test this PR**:
1. Launch the app
1. Go to `vanguard.com` and wait until the site loads
1. The privacy grade should be black
1. Click in the privacy grade
1. The privacy dashboard should have the toggle disabled (as in non changeable)
1. The privacy dashboard should show trackers found rather than blocked.
1. The privacy dashboard should show a warning saying the protections is disabled due to a breakage
1. The privacy dashboard should not show the Unprotected Sites and the Report Broken Site buttons
1. Click on the header of the dashboard, the privacy grade should open
1. Same warning message as before should show
1. Go to `cnn.com` and in the options menu disable privacy protection. Let the site load.
1. The privacy grade should be black
1. Click in the privacy grade
1. The privacy dashboard should have the toggle OFF but you should be able to turn it on.
1. The privacy dashboard doesn't show the prior warning message.
1. The privacy dashboard shows the Unprotected Sites and the Report Broken Site buttons
1. Go to `as.com` and let it load
1. The privacy grade should show a grade.
1. Click on the privacy grade, the privacy dashboard should have privacy protection enabled.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
